engine/src/main/java/com/datatorrent/stram/client/StramAgent.java

```diff
@@ -417,11 +418,23 @@ private StramWebServicesInfo retrieveWebServicesInfo(String appId)
       String appPath = response.getString("appPath");
       String user = response.getString("user");
       JSONObject permissionsInfo = null;
-      try (FSDataInputStream is = fileSystem.open(new Path(appPath, "permissions.json"))) {
+      Path permissionsPath = new Path(appPath, "permissions.json");
+      try (FSDataInputStream is = fileSystem.open(permissionsPath)) {
         permissionsInfo = new JSONObject(IOUtils.toString(is));
+      } catch (FileNotFoundException ex) {
+        // ignore and log messages if file is not found
+        if (LOG.isDebugEnabled()) {
+          LOG.debug("Permission information not present, file {} was not found", permissionsPath);
```

Comment:
This should be the log message regardless, why make distinction between info and debug in this case.